app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -160,14 +201,21 @@ public void onClick(View v) {
 
     private void addChapters(List<Chapter> chapterList) {
         Collections.sort(chapterList, mLocationComparator);
-        mSpinnerAdapter.clear();
-        mSpinnerAdapter.addAll(chapterList);
+        mChapterForCityName.clear();
+        mChapterForCityName.addAll(chapterList);
 
-        mChapterSpinner.setAdapter(mSpinnerAdapter);
+        autoCompleteSpinnerView.setAdapter(mChapterForCityName);
 
         if (mSelectedChapter != null) {
-            int pos = mSpinnerAdapter.getPosition(mSelectedChapter);
-            mChapterSpinner.setSelection(pos);
+            int pos = mChapterForCityName.getPosition(mSelectedChapter);
+            autoCompleteSpinnerView.setSelection(pos);
```

Comment:
I just got a crash here. Since AutoCompleteTextView is a TextView setSelection is actually the position of the cursor. We should do something else here. 

Details:
```
java.lang.IndexOutOfBoundsException: setSpan (296 ... 296) ends beyond length 2
                                                                                 at android.text.SpannableStringBuilder.checkRange(SpannableStringBuilder.java:1018)
                                                                                 at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:611)
                                                                                 at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:607)
                                                                                 at android.text.Selection.setSelection(Selection.java:76)
                                                                                 at android.text.Selection.setSelection(Selection.java:87)
                                                                                 at android.widget.EditText.setSelection(EditText.java:99)
                                                                                 at org.gdg.frisbee.android.onboarding.FirstStartStep1Fragment.addChapters(FirstStartStep1Fragment.java:210)
```